luda/editor/client/src/sequenceListView/type.ts

```diff
@@ -0,0 +1,34 @@
+import { Selection, XywhRect } from "namui";
+import { Track } from "../timeline/type";
+
+export type SequenceListViewState = {
+  layout: {
+    rect: XywhRect;
+    listWidth: number;
+  };
+  addingSequence: boolean;
+  renamingSequence: boolean;
```

Comment:
I think somebody can set `addingSequence = true`, `renamingSequence = true` at once. It shouldn't be, right? I suggest make it as enum.

Reply:
Fixed with 6b75133